planning.md

```diff
@@ -0,0 +1,12 @@
+# Seattle Theatre Site -- Planning
+
+Whenever moving forward on the project gets involved enough that I need to step back and create specific plans and strategies in order to proceed, I will use this document to try to get my ideas clear.
+
+## Current Issues
+
+1. I would like to pull out popup functionality to centralize it, but a couple things concern me:
+   - Where should popups be instantiated? (Thoughts: wherever they would otherwise be injected, for example, somewhere near the root of the application.)
```

Comment:
This is tricky, because you want to be able to define a popup anywhere, but still open it in a controlled place. Take a look at this component, whose demo opens the div on the body, rather than where the component is included:

http://www.qimingweng.com/react-modal-dialog/